packages/app/src/App.tsx

```diff
@@ -35,6 +40,7 @@ const App: FC<{}> = () => (
     <OAuthRequestDialog />
     <Router>
       <Root>
+        <Route key="login" path="/login" component={LoginPage} exact />,
```

Comment:
This little comma devil 👿  is making my view like this:
![Screenshot 2020-05-27 at 11 04 54](https://user-images.githubusercontent.com/3645856/82999860-e841d600-a009-11ea-8670-674ff6616cba.png)

Reply:
I have a fix